bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/PowermaxHandlerFactory.java

```diff
@@ -49,17 +50,13 @@
 
     private Map<ThingUID, ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
 
-    private @NonNullByDefault({}) SerialPortManager serialPortManager;
+    private SerialPortManager serialPortManager;
```

Comment:
You can now use final field modifier. For `discoveryServiceRegs` too.

Reply:
Done for serialPortManager.